### ISSUE TITLE
use postman in chrome found error : CSRF token missing

### ISSUE THREAD

**Issue by simdm** [krakenjs/lusca#73]
```
    app.use(lusca({
      csrf: {
        angular: true
      },
      xframe: 'SAMEORIGIN',
      hsts: {
        maxAge: 31536000, //1 year, in seconds
        includeSubDomains: true,
        preload: true
      },
      xssProtection: true
    }));
```

use postman in chrome found error : CSRF token missing

**Comment by lvarayut** [comment]
Did you figure it out? I'm facing the same issue.

**Comment by aredridel** [comment]
Did you send the token with your request?

**Comment by lvarayut** [comment]
Yes, I did. I put it in the header `Authorization: Bearer <token>`.

**Comment by danilodeveloper** [comment]
hello @lvarayut and @aredridel 
I'm facing the same issue here. 
I've tried to put the _csrf token in the message payload and in the header, both with same issue.

**Comment by jasisk** [comment]
> Yes, I did. I put it in the header Authorization: Bearer <token>.

Hello @lvarayut.

a csrf token is not an auth token—it won't work as a bearer token.

You'll want to set the `x-csrf-token` header to the csrf token (see [this test](https://github.com/krakenjs/lusca/blob/6c9a4663a58448497acd7e4aee7f35ae2f47e55d/test/csrf.js#L210) for an example). If you'd rather use a different value, simply pass a `header` value in with the options you use to configure csrf.

> I'm facing the same issue here.

Hey @danilodeveloper. It sounds like you may have a slight misconfiguration—perhaps load order? Are you loading lusca after you register your routes, by chance? Do you have body-parsing middleware before lusca?

**Comment by danilodeveloper** [comment]
Hi @jasisk!
I made a mistake, sorry. I created a route that was loaded after lusca.

Sorry again @jasisk :blush:

**Comment by jasisk** [comment]
No worries. Glad it worked out. :grinning:

**Comment by lvarayut** [comment]
@jasisk Perfect! Thanks for your response.

**Comment by dopplesoldner** [comment]
Hi guys.

I am having this issue as well and wasn't able to figure this out (perhaps due to my lack of experience with nodejs).

I've added an image to show what I am trying

![image](https://cloud.githubusercontent.com/assets/1665923/12567224/e4733454-c3b5-11e5-98e6-6091abefd565.png)

**Comment by ishaan-puniani** [comment]
Hi, I from where did you fetch this token. I want to have application to post data on other application. so i am looking for a method to fetch this token before making post request.

**Comment by saeidalidadi** [comment]
Hi, I have the same issue with postman

**Comment by vikasgautam6A29** [comment]
use "csrfmiddleware" : "your csrf token"  as a key-value pair, This solved my problem.
